app/subscriber/src/features/press-gallery/PressGallery.tsx

```diff
@@ -0,0 +1,47 @@
+import { DateFilter } from 'components/date-filter';
+import { FolderSubMenu } from 'components/folder-sub-menu';
+import { determineColumns } from 'features/home/constants';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { FlexboxTable, IContentModel, ITableInternalRow, Row } from 'tno-core';
+
+import * as styled from './styled';
+
+/** TODO: Didn't realize I had two different stories - this page will be done in the next ticket*/
+export const PressGallery: React.FC = () => {
+  const navigate = useNavigate();
+  // const [items, setItems] = React.useState<IContentModel[]>([]);
```

Comment:
Any reason to leave these commented out line?

Reply:
Was just for my next ticket (1728)!